lib/modules/datasource/hex/schema.ts

```diff
@@ -19,6 +20,13 @@ export const HexRelease = z
         inserted_at: z.string().optional(),
       }),
     ).refine((releases) => releases.length > 0, 'No releases found'),
+    retirements: z.record(
```

Comment:
Is this field optional?

Reply:
Yes

Reply:
Is it defined as optional in this zod schema?

Reply:
I think make it optional, even if it's meant to be there